### ISSUE TITLE
Unable to run any simulation

### ISSUE THREAD

**Issue by itsnishit** [edaplayground/eda-playground#79]
Whenever I try to save the code it shows this :
![a](https://user-images.githubusercontent.com/86912339/230773723-f1bb6159-0526-486c-9a37-6214a491abba.png)
Whenever I try to run the code it shows this :
![b](https://user-images.githubusercontent.com/86912339/230773730-8b93d32a-d2d5-41b9-9303-6ea1a77e28e0.png)

**Comment by kulkarniakshay02** [comment]
I am also getting the same error when I try to run specman e language related tests:

![image](https://user-images.githubusercontent.com/94942531/231775523-9997d2c2-08b7-4dd0-b0ff-21abb35f5938.png)

**Comment by edaplayground** [comment]
Did you try reloading the page?

**Comment by kulkarniakshay02** [comment]
Yes yes, tried reloading the page but, I get this error more often than the
times it works.

Akshay

**Comment by Daniyal-Tahsildar** [comment]
I have been facing the same issue as well. Crashes more times that it successfully runs. Kindly let me know how to resolve this. By the way, I have been working using UVM.

**Comment by edaplayground** [comment]
The problem is a bug. It tends to happen when you stay connected to a page and try to run (or do anything else) without first refreshing the page. I don't know any more than that. When I have time to look at it, I will be able to try to fix it. But I never have the time. Sorry for the inconvenience.

**Comment by edaplayground** [comment]
I'm afraid this is a bug. I don't know what causes it. I have not had the
time to look into it. I think it is worse at busy times of the day (the
working day in India). You could try reloading the page or waiting until a
quieter time.

Matthew
